changelogs/unreleased/gh-8204-select-offset-count-perf.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/box
+
+* Significantly improved the performance of `select` requests with the
+  `offset` specified and `count` requests. This prevents DoS conditions
```

Comment:
```suggestion
  `offset` specified and with `count` requests. This prevents DoS conditions
```

Reply:
The "`select` requests with the `offset` specified" and "`count` requests" are separated entities, performance is improved for each of them. It does not seem "with" can be applied here.

Please feel free to correct me.